doc/README.rst

```diff
@@ -0,0 +1,55 @@
+Documention
+!!!!!!!!!!!
+
+This directory contains source and tools to build the GA4GH schemas
+documentation.
+
+Building
+@@@@@@@@
+
+To build documentation, type::
+
+  make docs
+
+in the schemas root directory.  The resulting documentation will be in
+the ``schemas/build/html`` directory.
+
+.. note:: The ``tools/sphinx/generate_sphinx_docs.sh`` script is
+          obsolete.  The functionality is entirely subsumed by ``make
+          docs``, which also enables incremental updates.
+
+A `daily build
+<http://hgwdev.cse.ucsc.edu/~markd/ga4gh/documentation-pr/>`_ is also
+available at UCSC. 
+
+Changes are in progress to enable building the documentation at Read
```

Comment:
Checking -- do you want to leave this as "in progress" until all the i's are dotted for auto-building?